### PR TITLE
Update Moniker to include Software and Email

### LINKED ISSUE #3821: Update moniker which now supports email & software token
I can't find their documentation, but per the attached screenshots they now support Google Authenticator & email OTP

Screen Shot 2019-04-20 at 7 41 14 AM
Screen Shot 2019-04-20 at 7 41 58 AM

### PR DESCRIPTION
**Pull request by kmpoppe**
Close #3821.

### CODE REVIEW

**Review by Carlgo11 — APPROVED** [review]

File: _data/domains.yml [hunk 1]
```diff
@@ -320,9 +320,10 @@ websites:
     - name: Moniker
       url: https://www.moniker.com/
       img: moniker.png
-      tfa: No
-      twitter: monikersnap
-      facebook: MonikerLLC
+      tfa: Yes
+      software: Yes
+      email: Yes
```
Comment (Carlgo11) [inline _data/domains.yml:325]: Looks like they only support 2FA through software 2FA. Email seems to only be authorization.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — DISMISSED** [review]

**Review by jamcat22 — DISMISSED** [review]
Based on the doc and the screenshots in #3821, it looks like the "Authentication via e-mail prior to login" option does indeed trigger with each login, making it qualify for an `email: Yes` tag. @Carlgo11 any reason to think it's only authorization?

**Comment by Carlgo11** [comment]
I reviewed it so long ago but you're right @jamcat22, it does seem like it would trigger on each auth request. 
@kmpoppe could you revert that commit? Sorry 😅

**Comment by kmpoppe** [comment]
@Carlgo11, could you just commit to my branch, please? I don't quite read what you want it to show 😅

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**